tests/sentry/rules/processing/test_delayed_processing.py

```diff
@@ -42,12 +146,34 @@ def test_fetches_from_buffer_and_executes(self, mock_safe_execute):
         process_delayed_alert_conditions(mock_buffer)
 
         for project, rule_group_event_mapping in (
-            (self.project, rulegroup_event_mapping_one),
-            (project_two, rulegroup_event_mapping_two),
+            (self.project, self.rulegroup_event_mapping_one),
+            (self.project_two, self.rulegroup_event_mapping_two),
         ):
-            mock_safe_execute.assert_any_call(
-                apply_delayed,
-                project,
-                rule_group_event_mapping,
-                _with_transaction=False,
-            )
+            assert mock_apply_delayed.delay.call_count == 2
+
+    def test_apply_delayed(self):
+        mock_buffer = Mock()
+        mock_buffer.get_set.return_value = self.buffer_mapping.keys()
+        mock_buffer.get_hash.return_value = [
+            self.rulegroup_event_mapping_one,
+            self.rulegroup_event_mapping_two,
+        ]
+        for proj in [self.project, self.project_two]:
+            events = apply_delayed(proj, mock_buffer)
+            for event in events:
+                assert event in [self.event1, self.event2]
```

Comment:
right now the event id being returned is `3:2` which is the rule and group id from the buffer ... what